tests/TestCase/Action/AddActionTest.php

```diff
@@ -85,7 +85,7 @@ public function testActionPost()
             'Controller.initialize',
             ['priority' => 11],
             function ($event) {
-                $this->_controller->Flash = $this->getMockBuilder('Cake\Controller\Component\FlashComponent')
+                $this->_controller->Flash = $this->getMockBuilder(\Cake\Controller\Component\FlashComponent::class)
```

Comment:
`use` statements at the top would have been better. IDEs/Editors can do that for you.

Reply:
Not mine apparently (PHPStorm). I mean, it has a similar feature, but will also change it in the docblocks. I don't see a way to exclude those

Reply:
Yeah there's no way. It'll also add a `use` for things only mentioned in docblocks. You'll have to do it with your IDE I guess 😄

Reply:
I found it!!